tests/topotests/bgp_vpnv4_allocnh/test_bgp_vpnv4_allocnh.py

```diff
@@ -0,0 +1,386 @@
+#!/usr/bin/env python
+
+#
+# test_bgp_vpnv4_allocnh.py
+#
+# Copyright (c) 2022 by 6WIND
+#
+# Permission to use, copy, modify, and/or distribute this software
```

Comment:
Please replace this also to our new standard by defining a license with an SPDX-License tag. Check for examples in the latest master branch.

Reply:
done